zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -315,7 +315,7 @@ class Handlers(mustacheGenerator: ZipkinMustache, queryExtractor: QueryExtractor
         "parentId" -> span.parentId.filter(spanMap.get(_).isDefined).map(SpanId(_).toString),
         "spanName" -> span.name,
         "serviceNames" -> span.serviceNames.mkString(","),
-        "serviceName" -> span.serviceName,
+        "serviceName" -> span.serviceName.getOrElse(""),
```

Comment:
by contract, this shouldn't return null (as we shouldn't return spans w/o a service name) Any reason for this change?